python_ta/reporters/html_reporter.py

```diff
@@ -9,9 +10,11 @@
 from pylint.reporters.ureports.nodes import BaseLayout
 
 from .core import PythonTaReporter
-from .html_server import open_html_in_browser
+from .one_shot_server import open_html_in_browser
+from .persistent_server import start_server_once
 
 TEMPLATES_DIR = os.path.join(os.path.dirname(os.path.abspath(__file__)), "templates")
+PORT = None
```

Comment:
In general, avoid using global variables. In this particular case you can make this an instance attribute, as the same `HTMLReporter` is persisted across multiple runs of checking.